src/pal/src/misc/stackstring.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+#include "pal/malloc.hpp"
+
+template <UINT32 MEMSIZE>
+class StackString
```

Comment:
Woudln't it be more useful to actually make it a string instead of BYTE buffer? It would then make sense to add the char type as an additional template parameter.